scripts/js/commands/checkInternalLinksFromOtherRepo.ts

```diff
@@ -49,9 +49,9 @@ async function main() {
   const args = readArgs();
 
   const fileBatch = await FileBatch.fromGlobs(
-    ["docs/**/*.{ipynb,mdx}"],
+    ["{docs,learning}/**/*.{ipynb,mdx}"],
```

Comment:
I don't think this change is necessary. This script is solely intended for Qiskit Functions & the Tutorials team when developing new functionality in private. Generally, this script will probably need to be reworked because it's not yet handling how URLs should now have the `/docs` prefix (at least we haven't tested it)